static/sass/styles.scss

```diff
@@ -494,4 +494,45 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 /// can be removed once added upstream
 .is-paper {
   background-color: #f3f3f3;
-}
\ No newline at end of file
+}
+
+.p-department-cards__container {  
+  aspect-ratio: 2 / 3;
+  background-color: #fff;
+  margin-bottom: $spv--large;
+  margin-top: $spv--small;
+
+  @media screen and (max-width: ($breakpoint-large - 1)) {
```

Comment:
Why not the other way around? Just set it to `2 / 3` when min-width is above breakpoint large ?

Reply:
Done, thanks.